sambacc/config.py

```diff
@@ -237,7 +237,11 @@ def __init__(self, conf: GlobalConfig, iconfig: dict):
     def global_options(self) -> typing.Iterable[typing.Tuple[str, str]]:
         """Iterate over global options."""
         # Pull in all global sections that apply
-        gnames = self.iconfig["globals"]
+        try:
+            gnames = self.iconfig["globals"]
+        except KeyError:
+            # no globals section in the instance means no global options
+            return
```

Comment:
I say: `gnames = self.iconfig.get("globals", [])`

Reply:
This was an intentional choice in this case. It nets me a comment and an early return rather than having to fall thru to an empty loop or bother with an if statement.

Reply:
I now see it. Fair enough.